src/methods.ts

```diff
@@ -110,6 +113,55 @@ class JsonRpcMethods {
             })
             .catch((e) => nodeError(e, callback));
     }
+
+    getInstanceInfo(
+        blockHash: string,
+        index: bigint | number,
+        subindex: bigint | number,
+        callback: JSONRPCCallbackTypePlain
+    ) {
+        if (!isValidContractAddress(index, subindex)) {
+            return invalidParameterError(
+                'The provided contract address { index: ' +
+                    index +
+                    ', subindex: ' +
+                    subindex +
+                    '} is invalid',
+                callback
+            );
+        }
+        if (!isValidHash(blockHash)) {
+            return invalidParameterError(
+                'The provided blockHash [' + blockHash + '] is invalid',
+                callback
+            );
+        }
+
+        const getAddressInfoRequest = new GetAddressInfoRequest();
+        getAddressInfoRequest.setAddress(
+            JSON.stringify({ index: BigInt(index), subindex: BigInt(subindex) })
```

Comment:
This will allow for `string` values as input which should not be allowed.

Reply:
Removed it, and "improved the validation"